.github/workflows/javascript.yml

```diff
@@ -12,9 +12,11 @@ jobs:
         with:
           node-version: 18
 
-      - name: Install standard
+      - name: Install standard and React plugin for ESlint
         run: |
-          npm install standard --location=global
+          npm install standard@17.1.2 --location=global
+          npm install eslint@9.10.0 --location=global
```

Comment:
Hi, i noticed that "standard" has dependency eslint (with version ^8.41.x) and eslint-plugin-react (@7.36.1). So i am not sure if it is necessary to reinstall them again. I am curious what are your reasoning to reinstall them with latest version? 
see source repo: https://github.com/standard/standard/blob/master/package.json

Reply:
The issue that occurred in the last sprint was that there was a backwards-incompatible change in a new minor version of the React ESLint plugin. Since standard doesn't specify an exact dependency version, this broke the lint workflow. 

We discussed in the retro it would be better to pin these dependencies, so that the lint workflow doesn't break randomly whenever there's a new incompatible version of these dependencies.